### PR TITLE
Memory improvement when writing missing positions to pet

### PR DESCRIPTION
**Pull request by mmorgantaylor**
related to issue #211 and #233 - in CreateVariantIngestFiles, when writing missing positions to the pet tsv, we were looping through blocks of missing intervals twice, once to construct the pet rows, holding them all in memory, and again to write the pet rows. in this PR, we merge those steps into one: for each missing block, we write the pet lines to file as we loop through, avoiding the need to hold large blocks in memory.

note that this does not resolve the memory issues that originally prompted issues #211 and #233, but it's nonetheless a minor improvement that helps immensely when there are large numbers of missing locations.

### CODE REVIEW

File: scripts/variantstore/wdl/ImportGenomes.wdl [hunk 1]
```diff
@@ -168,7 +168,7 @@ task CreateImportTsvs {
         --mode GENOMES \
         -SNM ~{sample_map} \
         --ref-version 38
-        
+
```
Comment (ahaessly) [inline scripts/variantstore/wdl/ImportGenomes.wdl:171]: you can remove this class from the PR - only whitespace changes

**Review by ahaessly — APPROVED** [review]
👍 
there are actually 2 files that only have whitespace changes. they don't need to be in this PR

File: src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/PetTsvCreator.java [hunk 2]
```diff
@@ -287,18 +269,33 @@ private void setCoveredInterval(String variantChr, int start, int end) {
         return rows;
     }
 
-    public static List<List<String>> createMissingTSV(long start, long end, String sampleName) {
-        List<List<String>> rows = new ArrayList<>();
-
+    public void createMissingTSV(long start, long end, String sampleName) throws IOException {
```
Comment (kcibul) [inline src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/PetTsvCreator.java:272]: rename to `writeMissingEntry` or something else with `write`?

File: src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/PetTsvCreator.java [hunk 3]
```diff
@@ -287,18 +269,33 @@ private void setCoveredInterval(String variantChr, int start, int end) {
...
         for (long position = start; position <= end; position ++){
```
Comment (kcibul) [inline src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/PetTsvCreator.java:273]: nit -- remove space on postincrement operator (ie `position++`)

**Review by kcibul — APPROVED** [review]
looks great -- just small nits

**Pull request merged**